web/src/components/torrents/TorrentCreatorDialog.tsx

```diff
@@ -0,0 +1,366 @@
+              <div className="flex items-center justify-between">
+                <div className="space-y-0.5">
+                  <Label htmlFor="startSeeding">Add to qBittorrent</Label>
+                  <p className="text-sm text-muted-foreground">
+                    Add the created torrent to qBittorrent and start seeding. If disabled, only creates the .torrent file for download.
+                  </p>
+                </div>
+                <Switch
+                  id="startSeeding"
+                  checked={field.state.value}
+                  onCheckedChange={field.handleChange}
+                />
+              </div>
+            )}
+          </form.Field>
+
+          {/* Advanced Options */}
+          <Collapsible open={advancedOpen} onOpenChange={setAdvancedOpen}>
+            <CollapsibleTrigger asChild>
+              <Button
+                type="button"
+                variant="ghost"
+                className="w-full justify-between p-0 hover:bg-transparent"
+              >
+                <span className="text-sm font-medium">Advanced Options</span>
+                <ChevronDown
+                  className={`h-4 w-4 transition-transform ${advancedOpen ? "rotate-180" : ""}`}
+                />
+              </Button>
+            </CollapsibleTrigger>
+            <CollapsibleContent className="space-y-4 pt-4">
+              {/* Torrent Format */}
+              <form.Field name="format">
+                {(field) => (
+                  <div className="space-y-2">
+                    <Label htmlFor="format">Torrent Format</Label>
+                    <Select
+                      value={field.state.value}
+                      onValueChange={(value) => field.handleChange(value as TorrentFormat | "")}
+                    >
+                      <SelectTrigger id="format">
+                        <SelectValue placeholder="Auto (v1)" />
+                      </SelectTrigger>
+                      <SelectContent>
+                        <SelectItem value="v1">v1 (Compatible)</SelectItem>
+                        <SelectItem value="v2">v2 (Modern)</SelectItem>
+                        <SelectItem value="hybrid">Hybrid (v1 + v2)</SelectItem>
+                      </SelectContent>
+                    </Select>
+                    <p className="text-sm text-muted-foreground">
+                      v1 for maximum compatibility, v2 for modern clients, hybrid for both
+                    </p>
+                  </div>
+                )}
+              </form.Field>
+
+              {/* Piece Size */}
+              <form.Field name="pieceSize">
+                {(field) => (
+                  <div className="space-y-2">
+                    <Label htmlFor="pieceSize">Piece Size (bytes)</Label>
+                    <Input
+                      id="pieceSize"
+                      type="number"
+                      placeholder="Auto (leave empty)"
+                      value={field.state.value}
```

Comment:
Prohibit invalid values. Ideally, the steppers would only cycle through valid values. 

Probably needs a hardcoded list. There's _not that many_ valid values, and they're static also.

Reply:
> Leave empty for auto (recommended). Common: 16384, 32768, 65536, 131072, 262144

These are taken from upstream, and we can hardcode something for those. I'll double check that I got them right though.

Reply:
Done